zap/src/test/resources/org/zaproxy/zap/spider/parser/html/ImgElementsSpiderHtmlParser.html

```diff
@@ -14,8 +14,10 @@
 <img src="">
 <img src="/img/absolute">
 <img src="ftp://img.example.com/">
-<img longdesc="/relative/longdesc">
```

Comment:
Why this change?

Reply:
To keep it consistent with the comment here: https://github.com/zaproxy/zaproxy/pull/7159#discussion_r830386703

Reply:
With leading slash it’s actually absolute (from server root).

Reply:
Ha! I was thinking 'relative' as opposed to supplying the full URL, but I'll accept I'm wrong on this one :P